### PR TITLE
1529_dropdown_examples(size override for dropdown expand)

### PR DESCRIPTION
**Pull request by hellokitties**
https://github.com/jdi-testing/jdi-light/issues/1529

### CODE REVIEW

**Comment by mf-code** [comment]
[4428] - https://transfer.sh/hu0UG/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4428] - https://transfer.sh/cYKQj/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4428] - https://transfer.sh/syMJw/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4428] - Allure report on Netlify: https://5e17435daca58ad3b9901d7a--musing-swartz-6fc060.netlify.com

File: jdi-light-html-tests/src/test/java/io/github/epam/example/complex/DropdownExampleTests.java [hunk 1]
```diff
@@ -58,7 +58,6 @@ public void checkValuesTest() {
 
     @Test
     public void dropdownTestSize() {
-        colors2.expand();
         colors2.assertThat().notEmpty();
         colors2.assertThat().size(5);
     }
```
Comment (dexguitar) [inline jdi-light-html-tests/src/test/java/io/github/epam/example/complex/DropdownExampleTests.java]: Please ensure you have a blank line at the end of all files you modified

File: jdi-light/src/main/java/com/epam/jdi/light/elements/complex/dropdown/DropdownExpand.java [hunk 2]
```diff
@@ -112,5 +116,4 @@ public boolean isDisplayed() {
     public boolean isEnabled() {
         return value().isEnabled();
     }
-
 }
```
Comment (dexguitar) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/complex/dropdown/DropdownExpand.java]: Please ensure you have a blank line at the end of all files you modified

File: jdi-light-html-tests/src/test/java/io/github/epam/example/complex/DropdownExampleTests.java (hunk 1 above)
Comment (hellokitties) [reply jdi-light-html-tests/src/test/java/io/github/epam/example/complex/DropdownExampleTests.java]: have a blank line

File: jdi-light/src/main/java/com/epam/jdi/light/elements/complex/dropdown/DropdownExpand.java (hunk 2 above)
Comment (hellokitties) [reply jdi-light/src/main/java/com/epam/jdi/light/elements/complex/dropdown/DropdownExpand.java]: have a blank line in the end of file

**Pull request merged**